desktop/cmp/input/DateInput.ts

```diff
@@ -483,6 +483,7 @@ const cmp = hoistCmp.factory<DateInputModel>(({model, className, ...props}, ref)
                     selectOnFocus: props.selectOnFocus,
                     inputRef: model.inputRef,
                     ref: model.textInputRef,
+                    testId: getTestId(props, 'dateTextInput'),
```

Comment:
Here we're *only* installing a generated variant, which seems weird.  

Let's put the unmodified testId here, since it's the primary comp users interact with and has the `xh-date-input` className.  But let's install generated sub-testIds on the clear and show-picker buttons above